### ISSUE TITLE
[Research] "Fabric" library in research should be renamed to "Fluent UI"

### ISSUE THREAD

**Issue by melanierichards** [WICG/open-ui#204]
…in order to match current reality of Microsoft design systems

**Comment by melanierichards** [comment]
Pinging a couple Open UI members from Fluent UI to find out if there's any qualms/gotchas with doing so: @assuncaocharles @levithomason @khmakoto

**Comment by chrisdholt** [comment]
> Pinging a couple Open UI members from Fluent UI to find out if there's any qualms/gotchas with doing so: @assuncaocharles @levithomason @khmakoto

I think this would be preferred. While my contributions to Open UI are within the scope of FAST, I am also one of the core contributors to Fluent UI Web Components :). Considering the links direct to a Fluent name-spaced site and it's been broadly noted that Fabric is now "Fluent UI", I think this is overdue. While there is quite a bit of work happening in the Fluent UI space, I can only see a positive by ensuring the name is up to date.

@levithomason can correct me if I'm wrong, but in terms of the goals of Open UI, I think updating this makes the most sense.

**Comment by assuncaocharles** [comment]
I believe it would preferred as well.

**Comment by khmakoto** [comment]
Yeah, I think Fluent UI should be the new name going forward.

**Comment by gregwhitworth** [comment]
@chrisdholt @nicholasrice @melanierichards can one of you pick this up?

**Comment by chrisdholt** [comment]
@khmakoto should we really have both Fluent UI and Fluent UI Northstar? I feel like Fluent UI should really have a singular perspective from a standpoint of Open UI goals. If the goal is ultimately to align to Open UI, then I think Fluent UI should likely bring a singular perspective there. Thoughts @assuncaocharles, @levithomason, others?

**Comment by chrisdholt** [comment]
@gregwhitworth pending the above question as it may be more involved if we feel we should have a single representation of Fluent rather than two.

**Comment by levithomason** [comment]
I don't think we should do any renaming here. When the research was done the two libraries were called Fabric and Stardust. They were two separate libraries with two different APIs and two different styles. They were also developed independently buy separate teams and at different times. 

They have since converged into a single effort called Fluent UI. However, the converged framework that came out of this effort is a new library which has new APIs and new styles. If anything, we could add the converged Fluent UI as a new source. However, we should not merge Fabric and Stardust.
